rjd_gfx_d3d11.h

```diff
@@ -613,6 +613,10 @@ struct rjd_result rjd_gfx_command_pass_draw(struct rjd_gfx_context* context, str
 
 struct rjd_result rjd_gfx_command_buffer_commit(struct rjd_gfx_context* context, struct rjd_gfx_command_buffer* cmd_buffer)
 {
+    RJD_ASSERT(cmd_buffer);
```

Comment:
Yeah I need to do a pass on all the functions and add asserts for the params. Thanks for adding these.